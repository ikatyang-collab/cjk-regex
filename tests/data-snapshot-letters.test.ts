import { snapshotCharset } from './utilities.js'
import * as cjk from '../src/index.js'

snapshotCharset('letters', cjk.letters())
