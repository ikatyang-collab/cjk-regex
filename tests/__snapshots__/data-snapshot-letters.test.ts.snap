// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot: letters 1`] = `
"
4352 (ᄀ, U+1100) - 4607 (ᇿ, U+11FF)
11904 (⺀, U+2E80) - 11929 (⺙, U+2E99)
11931 (⺛, U+2E9B) - 12019 (⻳, U+2EF3)
12032 (⼀, U+2F00) - 12245 (⿕, U+2FD5)
12293 (々, U+3005)
12295 (〇, U+3007)
12321 (〡, U+3021) - 12329 (〩, U+3029)
12344 (〸, U+3038) - 12347 (〻, U+303B)
12353 (ぁ, U+3041) - 12438 (ゖ, U+3096)
12445 (ゝ, U+309D) - 12447 (ゟ, U+309F)
12449 (ァ, U+30A1) - 12538 (ヺ, U+30FA)
12541 (ヽ, U+30FD) - 12543 (ヿ, U+30FF)
12549 (ㄅ, U+3105) - 12591 (ㄯ, U+312F)
12593 (ㄱ, U+3131) - 12686 (ㆎ, U+318E)
12704 (ㆠ, U+31A0) - 12735 (ㆿ, U+31BF)
12784 (ㇰ, U+31F0) - 12830 (㈞, U+321E)
12896 (㉠, U+3260) - 12926 (㉾, U+327E)
13008 (㋐, U+32D0) - 13054 (㋾, U+32FE)
13056 (㌀, U+3300) - 13143 (㍗, U+3357)
13312 (㐀, U+3400) - 19903 (䶿, U+4DBF)
19968 (一, U+4E00) - 40959 (鿿, U+9FFF)
43360 (ꥠ, U+A960) - 43388 (ꥼ, U+A97C)
44032 (가, U+AC00) - 55203 (힣, U+D7A3)
55216 (ힰ, U+D7B0) - 55238 (ퟆ, U+D7C6)
55243 (ퟋ, U+D7CB) - 55291 (ퟻ, U+D7FB)
63744 (豈, U+F900) - 64109 (舘, U+FA6D)
64112 (並, U+FA70) - 64217 (龎, U+FAD9)
65382 (ｦ, U+FF66) - 65391 (ｯ, U+FF6F)
65393 (ｱ, U+FF71) - 65437 (ﾝ, U+FF9D)
65440 (ﾠ, U+FFA0) - 65470 (ﾾ, U+FFBE)
65474 (ￂ, U+FFC2) - 65479 (ￇ, U+FFC7)
65482 (ￊ, U+FFCA) - 65487 (ￏ, U+FFCF)
65490 (ￒ, U+FFD2) - 65495 (ￗ, U+FFD7)
65498 (ￚ, U+FFDA) - 65500 (ￜ, U+FFDC)
94179 (𖿣, U+16FE3)
94194 (𖿲, U+16FF2) - 94198 (𖿶, U+16FF6)
110576 (𚿰, U+1AFF0) - 110579 (𚿳, U+1AFF3)
110581 (𚿵, U+1AFF5) - 110587 (𚿻, U+1AFFB)
110589 (𚿽, U+1AFFD) - 110590 (𚿾, U+1AFFE)
110592 (𛀀, U+1B000) - 110882 (𛄢, U+1B122)
110898 (𛄲, U+1B132)
110928 (𛅐, U+1B150) - 110930 (𛅒, U+1B152)
110933 (𛅕, U+1B155)
110948 (𛅤, U+1B164) - 110951 (𛅧, U+1B167)
127488 (🈀, U+1F200)
131072 (𠀀, U+20000) - 173791 (𪛟, U+2A6DF)
173824 (𪜀, U+2A700) - 178205 (𫠝, U+2B81D)
178208 (𫠠, U+2B820) - 183981 (𬺭, U+2CEAD)
183984 (𬺰, U+2CEB0) - 191456 (𮯠, U+2EBE0)
191472 (𮯰, U+2EBF0) - 192093 (𮹝, U+2EE5D)
194560 (丽, U+2F800) - 195101 (𪘀, U+2FA1D)
196608 (𰀀, U+30000) - 201546 (𱍊, U+3134A)
201552 (𱍐, U+31350) - 210041 (𳑹, U+33479)
"
`;
